scripts/github-changelog.php

```diff
@@ -62,7 +64,7 @@ function fetch_last_PR() {
     $last_prs = json_decode( $data, true );
     $merged_prs = array_filter( $last_prs, function ( $pr ) { return $pr['merged_at'] ?? ''; } );
 
-    return $merged_prs[ 0 ];
+    return array_values( $merged_prs )[ 0 ];
```

Comment:
This will prevent getting `NULL` values because [array_filter](https://www.php.net/manual/en/function.array-filter.php) will preserve array keys.